universal-login-commons/src/core/services/EmailService.ts

```diff
@@ -0,0 +1,40 @@
+import nodemailer from 'nodemailer';
+import {MailOptions} from 'nodemailer/lib/json-transport';
+import {CannotSendEmail} from '../..';
+import Mail from 'nodemailer/lib/mailer';
+
+export class EmailService {
+  private transporter: Mail;
+  constructor() {
+    this.transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: 'testowyunilogin@gmail.com',
+        pass: 'password :)',
```

Comment:
These parameters should be taken from constructor

Reply:
Done